src/steps/finding/converter.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  createIntegrationEntity,
+  Entity,
+  parseTimePropertyValue,
+} from '@jupiterone/integration-sdk-core';
+
+import { Entities } from '../constants';
+import { Finding, FindingState } from '../../types';
+
+export function getFindingKey(id: string) {
+  return `probely_finding:${id}`;
+}
+
+// TODO: might need to be looked at again in the future.
+// From their API docs:
+// string (findingSeverity)
+// Enum: 10 20 30
+// Filter by finding severity
+function severityToImpact(severityValue: number): string {
+  if (severityValue === 10) {
+    return 'low';
+  } else if (severityValue === 20) {
+    return 'medium';
+  } else if (severityValue === 30) {
+    return 'high';
+  }
+  return 'unknown';
+}
+
+export function createFindingEntity(finding: Finding): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: finding,
+      assign: {
+        _key: getFindingKey(`${finding.target.id}:${finding.id}`),
```

Comment:
Minor nitpick but I'd probably change the signature of `getFindingKey` to accept targetId and findingId as parameters, seems a bit easier on the eyes.